build-system/tasks/presubmit-checks.js

```diff
@@ -390,6 +390,19 @@ var forbiddenTerms = {
       'src/service/url-replacements-impl.js',
     ]
   },
+  'expandStringSync\\W': {
```

Comment:
Will do, what is the difference between the `forbiddenTermsSrcInclusive` and `forbiddenTerms`?